src/openads/domain/position/PositionNotVisibleException.js

```diff
@@ -0,0 +1,8 @@
+export default class PositionNotVisibleException extends Error {
```

Comment:
We should agree in a name convention for our Custom Error classes. Sometimes we use _Exception_ subfix (like in PositionNotVisibleException), sometimes we do not (like in PositionAlreadyExists).

Reply:
Right. I kept in mind this subject to be spoken in some moment, maybe is now the right moment ;)

Reply:
We agreed to cover this topic in next iteration.